python/desc/sims/GCRCatSimInterface/InstanceCatalogWriter.py

```diff
@@ -117,6 +182,21 @@ def write_catalog(self, obsHistID, out_dir='.', fov=2):
         knots_name = 'knots_cat_%d.txt' % obsHistID
         #agn_name = 'agn_cat_%d.txt' % obshistid
 
+	# SN Data
+        snDataDir = '../../../../data'
```

Comment:
Can you replace this with
```
os.path.join(getPackageDir('sims_GCRCatSimInterface'), 'data')
```
where `getPackageDir` can be imported from `lsst.utils`.

That way, the code will be able to find the `csv` files, no matter where it is run from (assuming everything has been set up properly via EUPS)